tests/framework/cli/test_cli_hooks.py

```diff
@@ -108,3 +120,9 @@ def test_kedro_cli_should_invoke_cli_hooks_from_plugin(
             f"Before command `{command}` run for project {fake_metadata}"
             in result.output
         )
+
+        # 'pipeline list' and 'info' aren't actually in the click structure and
+        # return exit code 2 ('invalid usage of some shell built-in command')
```

Comment:
Do you happen to know why these command aren't in the click structure? Should they be?

Reply:
So I think the original test was just checking the command was captured, telemetry sends it even if it's actually invalid. I think it's more of an oversight than anything, but I would argue it would make sense to actually switch telemetry to the new one as we could track what the most common failed commands are.